### PR TITLE
[TIMOB-24807] Missing APIs in documentation

### PR DESCRIPTION
**Pull request by ewanharris**
JIRA: https://jira.appcelerator.org/browse/TIMOB-24807

I added the unimplemented properties that I believe are unimplemented.

Do I also need to push an updated apidoc/Titanium contents to the repo?

Verification:

- Run `node ti_win_yaml.js`, you should see files for SearchBar, and TCP.

### CODE REVIEW

**Comment by infosia** [comment]
@ewanharris 

> Do I also need to push an updated apidoc/Titanium contents to the repo?

Yes please. Note that yaml for `UDP` is located under `WindowsOnly` folder because it's for Windows only.

**Comment by ewanharris** [comment]
Ah thanks! Completely missed that, edited my comment to reflect that. I've updated the folder locally, it's a big diff so I'm going to pull up a list of commits that I can find that relate to addition/removal of properties to make sure it's all reflected correctly

**Comment by ewanharris** [comment]
Ok so this kinda got a bit beyond scope of the initial ticket. To summarise this PR now includes the following:

- Remove unimplemented notes for SearchBar, UDP, and TCP 1b95f06a9ad3e1db796c285f0812bba2b2e6250a
- Add unimplemented APIs in SearchBar 710c621cd17d50b7e623fe60f5b6debf3d858389
- Update ti_win_yaml.js to take constant definitions into consideration, and blacklist all namespaces in Titanium 1a6a26d9d8211cbee52517575e7cca3e7ab71327
- Remove unimplemented in Contacts, and add unimplemented in tableview e26d933a88df5fe9cfd0e93293a616a7aff5461b
- Update the whitelisted events 4e2a91d2e738574e1dd82b7993c3f7c3e7657590
- Update the apidocs that already existed 855410c
- Add newly generated apidocs cdd5ba2
- Formatting and cleanup of ti_win_yaml.js 3fab4a2


As this script has been running on jenkins, I grabbed the latest dist from the master branch and ran a diff of the two, this is ultimately what my changes really add. It can be found [here](https://gist.github.com/ewanharris/ba4395253fd6beb649af8adacbbf08af)


If it is preferred for this to be broken down into separate PRs to make it easier for review let me know

File: apidoc/Titanium/Blob.yml [hunk 1]
```diff
@@ -38,4 +38,16 @@ methods:
...
+    platforms: [windowsphone]
+  - name: imageAsResized
+    platforms: [windowsphone]
+  - name: imageAsThumbnail
+    platforms: [windowsphone]
+  - name: imageWithAlpha
+    platforms: [windowsphone]
+  - name: imageWithRoundedCorner
+    platforms: [windowsphone]
+  - name: imageWithTransparentBorder
```
Comment (infosia) [inline apidoc/Titanium/Blob.yml]: `imageWithAlpha`, `imageWithRoundedCorner` and `imageWithTransparentBorder` are not implemented. `Blob::transformImage` does not cover them.

**Comment by infosia** [comment]
@ewanharris please review my comments above, once it is addressed this should be ready for merge.

**Comment by ewanharris** [comment]
@infosia Sorry about that got wrapped up with 6.1.1. Pushed an update with the updated files due to  properties being implemented since I last visited this

**Comment by infosia** [comment]
I think this is ok to merge for now because this is kind of `Story` task. @ewanharris Do you think we need QE-testing for this?

**Comment by ewanharris** [comment]
@infosia Sorry I somehow wrote out a comment and never posted it. I agree that I don't think this needs to be put into qe-testing

**Pull request merged**